airbyte-commons-server/src/main/java/io/airbyte/commons/server/handlers/SourceDefinitionsHandler.java

```diff
@@ -152,11 +152,7 @@ public SourceDefinitionReadList listLatestSourceDefinitions() {
   }
 
   private List<StandardSourceDefinition> getLatestSources() {
-    try {
-      return githubStore.getLatestSources();
-    } catch (final InterruptedException e) {
-      throw new InternalServerKnownException("Request to retrieve latest destination definitions failed", e);
-    }
+    return remoteOssCatalog.getSourceDefinitions();
```

Comment:
Do we need to set up micronaut caching on the server app for this to get the latest values? `getRemoteDefinitionCatalog` is marked as cacheable, which I believe by default if there's no configuration means that we'll only retrieve the values once and cache the results essentially until server restart. 

airbyte-cron has an example for how the config is modified there to cache the results for 15 seconds:

https://github.com/airbytehq/airbyte-platform/blob/016e80f82a7027af7840ba20689f57fabb853c8c/airbyte-cron/src/main/resources/application.yml#L4-L6

Reply:
@pedroslopez I think by definition its cached as `RemoteDefinitionsCatalog:getRemoteDefinitionCatalog` is tagged with `@cachable`

Reply:
Yes, but my concern is that I believe this default means that the cache never expires, so in practice we would only fetch what the latest version is _once_, and then always reuse the same values and never make another request to fetch latest values again.